src/commands/registerCommands.ts

```diff
@@ -60,6 +62,15 @@ export function registerCommands(): void {
         ext.actions.refreshFocusTree(node);
     });
 
+    registerCommand('azureTenant.refresh', async (context, node?: ResourceGroupsItem) => {
+        await handleAzExtTreeItemRefresh(context, node); // for compatibility with v1.5 client extensions
```

Comment:
Are any of the v1.5 extensions going to be interacting with the new tenant tree?

Reply:
Not exactly sure if there will be any direct interactions but Alex did mention in the issue he created (https://github.com/microsoft/vscode-azureresourcegroups/issues/849) he wanted it compatible with the non v2 extensions which is why I kept this in.